### PR TITLE
Remove TimeLogger class

### PR DESCRIPTION
**Pull request by DanielKamkha**
TimeLogger functionality was replaced by a build system solution.
Therefore TimeLogger class and its associated command line parameter and
CMake variables are obsolete and should be removed.

Resolves: #50

### CODE REVIEW

File: examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake [hunk 1]
```diff
@@ -31,16 +31,10 @@ function(_generate)
         unset(_build_local_gluecodium)
     endif()
 
-    if(DEFINED ENV{GLUECODIUM_TIME_LOGGING})
-        set(_gluecodium_time time -o "${APIGEN_BUILD_OUTPUT_DIR}/gluecodium-time.log")
-    else()
-        unset(_gluecodium_time)
-    endif()
```
Comment (Dschoordsch) [inline examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake]: This is external (i.e. build system) time measuring using the command line tool "time", please keep it.

File: examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake [hunk 2]
```diff
@@ -31,16 +31,10 @@ function(_generate)
...
-
     execute_process(
         COMMAND ${CMAKE_COMMAND} -E make_directory ${APIGEN_OUTPUT_DIR} # otherwise java.io.File won't have permissions to create files at configure time
         COMMAND ${CMAKE_COMMAND} -E make_directory ${APIGEN_COMMON_OUTPUT_DIR}
-        COMMAND ${_gluecodium_time} ${APIGEN_GLUECODIUM_GRADLE_WRAPPER} ${_build_local_gluecodium} -Pversion=${APIGEN_GLUECODIUM_VERSION} run --args=${APIGEN_GLUECODIUM_ARGS}
```
Comment (Dschoordsch) [inline examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake]: please keep

File: examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake (hunk 1 above)
Comment (DanielKamkha) [reply examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake]: Done

File: examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake (hunk 2 above)
Comment (DanielKamkha) [reply examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake]: Done

**Pull request merged**